packages/workspace/src/command-line/utils.ts

```diff
@@ -176,7 +176,10 @@ function printArgsWarning(options: NxArgs) {
         '',
         output.colors.gray(
           'Learn more about checking only what is affected: '
-        ) + 'https://nx.dev/guides/monorepo-affected.',
+        ) +
+          'https://nx.dev/latest/angular/cli/affected, ' +
```

Comment:
Use this URL: https://nx.dev/latest/core-concepts/affected

It will ask you what framework you want to use when you open it

Reply:
This URL ist also leading to the last selected framework... I will delete this PR.